File: Exams/Exam-02/Exam-02-1/Code/MoneyBankAPI/Models/Account.py
from sqlalchemy import Table, Column
from sqlalchemy.sql.sqltypes import Integer, String, DateTime,DECIMAL
from Config.db import engine, meta_data

Accounts = Table("Accounts", meta_data,
                 Column("id", Integer, primary_key=True, nullable=True),
                 Column("AccountType", String(1), nullable=False),
                 Column("CreationDate", DateTime, nullable=False),
                 Column("AccountNumber", String(10), nullable=False),
                 Column("OwnerName", String(100), nullable=False),
                 Column("BalanceAmount", DECIMAL(18,2), nullable=False),
                 Column("OverdraftAmount", DECIMAL(18,2), nullable=False))

meta_data.create_all(engine)

File: Exams/Exam-02/Exam-02-1/Code/MoneyBankMVC/Models/Account.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MoneyBankMVC.Models
{

    [Table("accounts")]
    public class Account
    {
        
        //[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key]
        public int Id { get; set; }

        [Required] [StringLength(1)] public string AccountType { get; set; } = null!;

        [Required]
        [Column(TypeName = "datetime")]
        public DateTime CreationDate { get; set; }

        [Required] [StringLength(10)] public string AccountNumber { get; set; } = null!;

        [Required] [StringLength(100)] public string OwnerName { get; set; } = null!;

        [Required]
        [Column(TypeName = "decimal(18,2)")]
        public decimal BalanceAmount { get; set; }

        [Required]
        [Column(TypeName = "decimal(18,2)")]
        public decimal OverdraftAmount { get; set; }
    }
}